scripts/uberenv/vcpkg_ports/hdf5/xCONTROL

```diff
@@ -0,0 +1,5 @@
+Source: hdf5
+Version: 1.10.5-8
```

Comment:
Note (for future): ``VisIt`` currently only supports hdf5 1.8.x, so we should modify this recipe, if possible, to use a ``VisIt`` compatible version.

This can be done is a separate/future PR.

@cyrush -- can you please confirm / weigh in on this?

Reply:
yes that is true - however vcpkg may not support HDF5 1.8. Not sure there is a cmake-based build that folks use for 1.8.

Reply:
I don't think vcpkg supports HDF5 v1.8.